### ISSUE TITLE
PANIC: not implemented: Arrays are not supported for sqlite.

### ISSUE THREAD

**Issue by timsuchanek** [prisma/prisma-client-js#705]
Hi Prisma Team! My Prisma Client just crashed. This is the report:
## Versions

| Name     | Version            |
|----------|--------------------|
| Node     | v12.16.3           | 
| OS       | darwin             |
| Prisma   | 2.0.0-alpha.1246   |

## Logs
```
  prisma-client Client Version 2.0.0-alpha.1246  +0ms
  prisma-client Engine Version b6745bd0f67138b4b68e00f607c99483016924c8  +0ms
  prisma-client {
  prisma-client   engineConfig: {
  prisma-client     cwd: '/Users/tim/code/repros/raw-test/prisma',
  prisma-client     debug: false,
  prisma-client     datamodelPath: '/Users/tim/code/repros/raw-test/node_modules/.prisma/client/schema.prisma',
  prisma-client     prismaPath: undefined,
  prisma-client     generator: {
  prisma-client       name: 'client',
  prisma-client       provider: 'prisma-client-js',
  prisma-client       output: '/Users/tim/code/repros/raw-test/node_modules/@prisma/client',
  prisma-client       binaryTargets: [],
  prisma-client       config: {}
  prisma-client     },
  prisma-client     showColors: false,
  prisma-client     logLevel: undefined,
  prisma-client     logQueries: undefined,
  prisma-client     flags: [],
  prisma-client     clientVersion: '2.0.0-alpha.1246'
  prisma-client   }
  prisma-client }  +0ms
  prisma-client Prisma Client call:  +3ms
  prisma-client prisma.raw(SELECT * FROM User WHERE id = $1, [[1]])  +0ms
  prisma-client Generated request:  +2ms
  prisma-client mutation {
  prisma-client   executeRaw(
  prisma-client     query: "SELECT * FROM User WHERE id = $1"
  prisma-client     parameters: "[[1]]"
  prisma-client   )
  prisma-client }
  prisma-client   +0ms
  plusX Execution permissions of /Users/tim/code/repros/raw-test/node_modules/.prisma/client/query-engine-darwin are fine  +0ms
```

**Comment by pimeys** [comment]
We should not panic in Quaint for unsupported types. Lately with my MSSQL changes the AST visitor code can return results, which means no need to panic anymore, which then means reasonable errors.

**Comment by pimeys** [comment]
Fixed in prisma/prisma-engines#827
